content/tutorials/unityxrpk-networked.mdx

```diff
@@ -0,0 +1,58 @@
+a popular networking library for Unity. Luckily, Photon has a free tier, so you can download the project, add your own Photon app ID (which you can get by [registering your app](https://developer.photonengine.com/dashboard/)) and witness XR Package networking in action!
+
+It's important to mention you can use _any_ networking solution you like, as long as it's _supported in the WebGL runtime_ (which unfortunately, as of the time of writing, excludes the equally popular Normcore.)
+
+You'll notice the project is very similar to the Sample Package you've encountered in the previous tutorial. In fact, the only difference is the inclusion and implementation of Photon (which also involves a minor refact to the Cubes prefab - it is now instantiated when the first user has joined, or created, the room).
+
+Before we start, we recommend you acquaint yourself with [Photon/PUN fundamentals](https://doc.photonengine.com/zh-cn/pun/v2/demos-and-tutorials/pun-basics-tutorial/intro).
+It's important to note this is just an example networking implementation - the XR Package-critical step will be described in step 2.
+
+## 0. Preparation
+
+As mentioned, you'll have to add your own App ID to Photon - this is a unique identifier for your app, and is used to identify your app when connecting to a specific Photon server. It's free to get (see above link), and once you have it, you'll need to add it to the project.
+Go to `Window/Photon Unity Networking/Highlight Server Settings"` and add your App ID. You should be good to go - hitting `Play` will demonstrate the stages of connecting to a server in the console and instantiate the Cubes prefab.
+
+## 1. General Scene Changes
+
+a. Note that the Left Controller and Right Controller scene objects now have additional components added to them: a Photon View and a Photon Transform View.
+These are built-in compoenents that, without any code, let us sync the position and rotation of the controller with the position and rotation of each controller in the Unity scene (assuming we're networked - which we'll discuss in the next step).
+
+![](./unity_xrpk_networked/1.png)
+
+It's important to note that even though were are not displaying the controllers graphically (since, when working with multi apps, we can see the controllers represented in our runtime environment like SteamVR), we are still able to sync the position and rotation of the controllers - furthermore this is critical to ensure that the cubes respond to each controller across _every other networked instance of the scene_, and not just in ours.
```

Comment:
Side note: This is awesome that networking in a Unity scene works this way compared to how we move around objects in our other multi-apps! Makes way more sense to network the interactions and not the physics.

Reply:
probably the nicest thing about Unity, indeed. so many systems - out of the box!